mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ProxyOperation.java

```diff
@@ -172,15 +171,15 @@ public ModelNode execute(ExpressionResolver expressionResolver, ModelNode operat
         public ModelNode execute(ExpressionResolver expressionResolver, ModelNode operation, ModClusterServiceMBean service) {
             boolean enabled = service.enable();
 
-            return new ModelNode().get(ModelDescriptionConstants.RESULT).set(enabled);
+            return new ModelNode(enabled);
```

Comment:
Most of these enums don't override 'apply' and thus don't configure an OperationDescription that describe any reply at all. OTOH that maybe gives you freedom to change the undocumented reply type. OTOH, it's not a correct description.

Reply:
Yeah, that was not the intention. Opened https://issues.jboss.org/browse/WFLY-11246 to address these.

Reply:
Fixed now in master.